### ISSUE TITLE
Linting whitespace between Jinja brackets

### ISSUE THREAD

**Issue by drewburr** [ansible-community/ansible-lint#1019]
##### Summary

Add linting for whitespace between Jinja curly brackets.

##### Issue Type

- Feature Idea

##### Additional Information

I'm not sure how large of an impact this would have, but I feel it makes sense to include since Ansible complains when there is a space in the 2nd curly bracket (See examples).

**Good example:**

```yaml
- debug:
    msg: "{{ test_var }}"
``` 

```console
TASK [debug]
ok: [localhost] => {
    "msg": "THIS IS A TEST"
}
```

**Bad example:**

```yaml
- debug:
    msg: "{{ test_var } }"
```

```console
TASK [debug]
fatal: [localhost]: FAILED! => {"msg": "template error while templating string: unexpected '}'. String: {{ test_var } }"}
```

**Other example**

This example runs successfully, however I don't believe this would be intentional.

```yaml
- debug:
    msg: "{ { test_var }}"
```

```console
TASK [debug]
ok: [localhost] => {
    "msg": "{ { test_var }}"
}
```
